tests/utils/LinearAlg2DTest.cpp

```diff
@@ -136,57 +133,63 @@ TEST_P(GetAngleTest, GetAngle)
 }
 
-INSTANTIATE_TEST_CASE_P(GetAngleInstantiation, GetAngleTest, testing::Values(
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(100, 1), 180), //Almost straight line.
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(100, 0), 180), //Completely straight line.
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, -100), 315), //-45 degrees.
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(0, -100), 270), //-90 degrees.
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(0, 100), 90), //Straight angle.
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, 1), 0), //Almost straight back.
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, -1), 360), //Almost straight back but the other way around.
-    GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, 0), 0) //Completely straight back.
-));
+INSTANTIATE_TEST_CASE_P(
+  GetAngleInstantiation,
+  GetAngleTest,
+  testing::Values(GetAngleParameters(Point(-100, 0), Point(0, 0), Point(100, 1), 180), // Almost straight line.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(100, 0), 180), // Completely straight line.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, -100), 315), //-45 degrees.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(0, -100), 270), //-90 degrees.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(0, 100), 90), // Straight angle.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, 1), 0), // Almost straight back.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, -1), 360), // Almost straight back but the other way around.
+                  GetAngleParameters(Point(-100, 0), Point(0, 0), Point(-100, 0), 0) // Completely straight back.
+                  ));
 
 TEST(GetAngleTest, GetAngleLeftAABTest)
 {
-    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(100, 0)); //Any output is allowed. Just don't crash!
+    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(100, 0)); // Any output is allowed. Just don't crash!
 }
 
 TEST(GetAngleTest, GetAngleLeftABBTest)
 {
-    LinearAlg2D::getAngleLeft(Point(0, 0), Point(100, 0), Point(100, 100)); //Any output is allowed. Just don't crash!
+    LinearAlg2D::getAngleLeft(Point(0, 0), Point(100, 0), Point(100, 100)); // Any output is allowed. Just don't crash!
 }
 
 TEST(GetAngleTest, GetAngleLeftAAATest)
 {
-    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(0, 0)); //Any output is allowed. Just don't crash!
+    LinearAlg2D::getAngleLeft(Point(0, 0), Point(0, 0), Point(0, 0)); // Any output is allowed. Just don't crash!
 }
 
 TEST(PointIsLeftOfLineTest, LeftOfLine)
 {
     constexpr short actual = 1;
-    const Point p(0, 10);  //    ^
-    const Point a(10, 0);  //  . |
+    const Point p(0, 10); //    ^
+    const Point a(10, 0); //  . |
     const Point b(10, 20); //    |
```

Comment:
This broke the alignment in the comments. I suppose that's why those extra spaces were there.

Reply:
Then I will add them to the other side of the `//`